jeos-firstboot-enroll

```diff
@@ -231,11 +250,11 @@ enroll_post() {
 enroll_tpm_and_fido() {
 	# For now is a first step before moving into fde-tools
 	local fde_cfg='/etc/sysconfig/fde-tools'
-	if [ -e "$cfg" ]; then
-		. "$cfg"
+	if [ -e "$fde_cfg" ]; then
```

Comment:
should be part of the other PR?

Reply:
was a mistake when merging diskencrypt and enroll. folded into parent commit now